README.md

```diff
@@ -2,9 +2,95 @@
 
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-MicroCeph is snap-deployed Ceph with built-in clustering.
+<p align="center">
+[MicroCeph](https://snapcraft.io/microceph) is snap-deployed Ceph with built-in clustering.
```

Comment:
Afaik once in html context, markdown won't process md syntax. Would need to spell out the link with html:

```
<p align="center">
<a href="https://snapcraft.io/microceph"><MicroCeph</a> is snap-deployed Ceph with built-in clustering.
</p>
```

Reply:
Ack, this behaviour is mixed amonst various md renders.